app/assets/javascripts/student_profile/AcademicSummary.js

```diff
@@ -0,0 +1,548 @@
+                72
+              </td>
+              <td style={styles.core}>
+                108
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                88
+              </td>
+              <td style={styles.core}>
+                123
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                100
+              </td>
+            </tr>
+            <tr>
+              <td>
+                Developing Reading Fluency - Percentage
+              </td>
+              <td colSpan={12}>
+                {' '}
+              </td>
+              <td style={styles.core}>
+                85%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                73%
+              </td>
+              <td style={styles.core}>
+                92%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                84%
+              </td>
+              <td style={styles.core}>
+                93%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                84%
+              </td>
+              <td style={styles.core}>
+                95%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                91%
+              </td>
+              <td style={styles.core}>
+                97%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                93%
+              </td>
+              <td style={styles.core}>
+                96%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                91%
+              </td>
+              <td style={styles.core}>
+                97%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                93%
+              </td>
+              <td style={styles.core}>
+                98%
+              </td>
+              <td style={styles.strategic}>
+                {' '}
+              </td>
+              <td style={styles.intensive}>
+                95%
+              </td>
+            </tr>
+          </tbody>
+        </table>
+      </div>
+    );
+  }
+
+  render() {
+    return (
+      <div className="AcademicSummary">
```

Comment:
This is an existing bug, but should be `SummaryWithoutSparkline`, can you rename while you're in here?

Reply:
FYI, I changed this in the other file, GitHub is just confused.